tests/models/abiotic/test_abiotic_model.py

```diff
@@ -245,12 +260,17 @@ def test_generate_abiotic_model_bounds_error(
     caplog.clear()
 
     # Check whether model is initialised (or not) as expected
-    with raises:
-        _ = AbioticModel.from_config(
-            data=dummy_climate_data,
-            core_components=core_components,
-            config=config,
-        )
+    with (
+        patch_run_update("abiotic"),
+        patch_bypass_setup("abiotic") as mock_bypass_setup,
+    ):
```

Comment:
TIL this syntax exists! I've been nesting all my `with` blocks till now, which looks awful :see_no_evil: